CHANGELOG.md

```diff
@@ -18,6 +18,10 @@
   replaced by the `--` separator for extra arguments.
 - Changed the output of the `--version` command line parameter to include a list
   of supported snapshot data format versions for the firecracker binary.
+- The maximum number of virtio devices attached at the same time has increased
+  from 11 to 19, as Firecracker now supports 24 interrupts.
```

Comment:
This is confusing. The user would wonder why only 19 devices if 24 interrupts are supported. Let's avoid this and clearly state the gist of the change.
```suggestion
- Increased the maximum number of virtio devices from 11 to 19.
```

Reply:
Done